pkgs/development/python-modules/shiboken6/default.nix

```diff
@@ -47,6 +54,10 @@ stdenv'.mkDerivation rec {
     "-DBUILD_TESTS=OFF"
   ];
 
+  postPatch = ''
+    substituteInPlace cmake/ShibokenHelpers.cmake --replace '#!/bin/bash' "#!$SHELL"
+  '';
+
   # Due to Shiboken.abi3.so being linked to libshiboken6.abi3.so.6.5 in the build tree,
```

Comment:
you should also probably change libshiboken6.abi3.6.5.dylib to libshiboken6.abi3.6.6.dylib (haven't tested this on Darwin yet, but it's highly likely)

Reply:
(I super appreciate the review, by the way ❤️)